figures/alg_comparison.py

```diff
@@ -47,15 +47,17 @@
         model.fit(data.generate())
 
         # Plot learning curve.
-        ax.plot(model.time_hist, model.loss_hist,
+        # ax.plot(model.time_hist, model.loss_hist,
+        #         color=color, label=method, **plot_options)
+        ax.plot(model.loss_hist,
                 color=color, label=method, **plot_options)
         ax.set_title(data.name)
         ax.set_ylim(0, 1)
 
 # Format subplots
 axes[0].set_ylabel("loss")
 for ax in axes:
-    ax.set_xlabel("time (s)")
+    ax.set_xlabel("iterations")
```

Comment:
Why the change from time to iterations? Maybe add a TODO comment saying that we should have two rows of plots. The top row will be iterations and the bottom row will be time?

Honestly, the iteration count doesn't seem too informative...

Reply:
In person conversation. Will keep this.